Sleek/README.md

```diff
@@ -11,8 +11,6 @@
 ### Psycho
 ![Psycho Screenshot](psycho.png)
 
-### Deeper
```

Comment:
Why did you remove the colour scheme? Please add it again, I cannot select which commits are merged in a PR